tests/ttnn/unit_tests/operations/test_add.py

```diff
@@ -259,7 +259,7 @@ def test_prim_add(device, shape):
 
     input_tensor_a = ttnn.from_torch(torch_input_tensor_a, layout=ttnn.TILE_LAYOUT, device=device)
     input_tensor_b = ttnn.from_torch(torch_input_tensor_b, layout=ttnn.TILE_LAYOUT, device=device)
-    output_tensor = ttnn.prim.binary(input_tensor_a, input_tensor_b, ttnn.BinaryOpType.ADD, in_place=True)
+    output_tensor = ttnn.prim.binary(input_tensor_a, input_tensor_b, ttnn.BinaryOpType.ADD)
```

Comment:
need to find a way to use output_tensor arg and perform inlace instead, since we are passing BinaryOpType directly